README.md

```diff
@@ -137,3 +138,33 @@ Start the monitor:
 Close the monitor:
 
     {"monitor" {"action": "close"}}
+  
+## FAQ
+<b>1. In the Erlang/OTP 21, erlport may not be compiled, because the latest version was not reflected in the official Erlport GitHub.</b>
+
+  - SOL) Change the version of erlport manually in <b>mix.exs</b>. As of October, 2018, the "0.10.0" version worked. See this https://github.com/hdima/erlport
+    ```
+    defp deps do
+        [
+          {:erlport, "~> 0.10.0"}, # Choose the version.
+          {:poolboy, "~> 1.5"}
+        ]
+    end
+    ```
+  
+<b>2. Failed to fetch record for 'hexpm/poolboy' from registry (using cache)</b>
+
+  - SOL) This was a cache problem, remove the cache file and try again.
+    ```
+    rm "~/.hex/cache.ets"
+    ```
+    
+<b>3. When you run a client, if "what():  Connection refused." error occurs</b>
+  
+  - SOL) If you use your own server instance, changed the line 17 code const std::string host = "kurg.org"; to const std::string host = "127.0.0.1"; in example.cpp and try again.
```

Comment:
Do you think ``127.0.0.1`` should be the default?

Reply:
I think 127.0.0.1 should be the default given that users follow the tutorial in the README.
The reason for this is that the tutorial is supposed to use own server.

Reply:
Agreed, do you like to change the value to "127.0.0.1"? Don't feel obligated, I can do this as well.

Reply:
Okay!!